client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-videopress-loading-messages.ts

```diff
@@ -1,6 +1,7 @@
 import { useI18n } from '@wordpress/react-i18n';
+import { LoadingMessage } from './types';
```

Comment:
Nit: this should be `import type`.
```suggestion
import type { LoadingMessage } from './types';
```